app/Http/Controllers/Api/AirportController.php

```diff
@@ -111,4 +111,20 @@ public function distance(string $fromIcao, string $toIcao): AirportDistanceResou
             'distance' => $distance,
         ]);
     }
+
+    /**
+     * Search for airports in the database
+     *
+     * @param string $searchString
+     *
+     * @return AnonymousResourceCollection
+     */
+    public function search(Request $request): AnonymousResourceCollection
+    {
+        $this->airportRepo->resetCriteria();
+        $this->airportRepo->pushCriteria(app(RequestCriteria::class));
+        $airports = $this->airportRepo->paginate(null, ['id', 'icao', 'name', 'hub']);
```

Comment:
What about adding IATA code to searches too ?

`$airports = $this->airportRepo->paginate(null, ['id', 'icao', 'iata', 'name', 'hub']);`

Reply:
Good idea